File: app/src/main/java/fr/frogdevelopment/nihongo/dico/data/rest/Sentence.java
package fr.frogdevelopment.nihongo.dico.data.rest;

import java.io.Serializable;

import fr.frogdevelopment.nihongo.dico.to_delete.entity.Result;

public class Sentence implements Serializable {

    private static final long serialVersionUID = 1L;

    public Result japanese;
    public Result translation;

}

File: app/src/main/java/com/frogdevelopment/nihongo/dico/data/entities/Sentence.kt
package com.frogdevelopment.nihongo.dico.data.entities

import androidx.room.ColumnInfo
import androidx.room.Entity
import androidx.room.PrimaryKey

@Entity(tableName = "sentences")
class Sentence(
        @PrimaryKey(autoGenerate = true) val rowid: Int,
        @ColumnInfo(name = "japanese") val japanese: String,
        @ColumnInfo(name = "translation") val translation: String,
        @ColumnInfo(name = "linking") val linking: String)